apps/ui/src/composables/useSpaces.ts

```diff
@@ -107,7 +107,9 @@ export function useSpaces() {
           },
           filter
         );
-        explorePageSpaces.value.push(...spaces);
+
+        explorePageSpaces.value = [...(overwrite ? [] : explorePageSpaces.value), ...spaces];
```

Comment:
```suggestion
        explorePageSpaces.value = overwrite ? [] : [explorePageSpaces.value, ...spaces];
```

I think it's way more readable.